Makefile-rpm-ostree.am

```diff
@@ -149,6 +149,7 @@ BUILT_SOURCES += $(binding_generated_sources)
 install-rpmostree-hook:
 	install -d -m 0755 $(DESTDIR)$(bindir)
 	install -m 0755 -t $(DESTDIR)$(bindir) rpm-ostree
+	install -m 0755 rpm-ostree $(DESTDIR)$(bindir)/rpm-ostree-unpriv
```

Comment:
Maybe `$(DESTDIR)$(libexecdir)` instead? It's not really something we expect users to interact with.